### ISSUE TITLE
Unknown OVAL info should not be fatal to tui.

### ISSUE THREAD

**Issue by zapman449** [future-architect/vuls#529]
# Environment

kubernetes

## Vuls

vuls v0.4.1 5c84ebe

## OS
- Target Server: Debian 8.9
- Vuls Server: Debian 8.9

## Go
- Go version: 1.8.3

# Current Output

`[Oct 30 00:35:07] ERROR [localhost] Failed to fill OVAL information: OVAL for  is not implemented yet`

TL;DR: Ran vuls to check host and docker containers running on a kubernetes worker node.  Got lots of errors (mostly on continers/pods which are just go binaries).  Went to view the report with `vuls tui` and got the above error message.

This should be a WARNING not ERROR.  The rest of the report is not invalidated because vuls failed to discover the base distro of the container.

**Comment by kotakanbe** [comment]
The information of OVAL should be used in terms of detection accuracy.
So, instead of not using oval information as a warning, Vuls should be fixed to detect the distribution type and version.

The detection logic is here.
https://github.com/future-architect/vuls/blob/master/scan/debian.go#L58

Please paste the output of `scan --debug` to see the output of the commands.

**Comment by zapman449** [comment]
```
  stderr:
  err: exit status 126
[Oct 31 16:02:25] DEBUG [localhost] Executing... ls /etc/system-release
[Oct 31 16:02:25] DEBUG [localhost] execResult: servername:
  cmd: docker exec --user 0 5e25e44d1427 /bin/bash -c 'ls /etc/system-release'
  exitstatus: 126
  stdout: rpc error: code = 13 desc = invalid header field value "oci runtime error: exec failed: container_linux.go:247: starting container process caused \"exec: \\\"/bin/bash\\\": stat /bin/bash: no such file or directory\"\n"
```

There's a sample of several thousand lines of output.  If I can be more specific, let me know

**Comment by kotakanbe** [comment]
```
stat /bin/bash: no such file or directory
```
`bash` must be installed on scan target server.

see https://github.com/future-architect/vuls#misc
```
Does Vuls accept SSH connections with fish-shell or old zsh as the login shell?
No, Vuls needs a user on the server for bash login. see also #8
```

**Comment by kotakanbe** [comment]
@zapman449 
Debian seems to be able to scan even /bin/sh, so I'll fix it.

**Comment by kotakanbe** [comment]
@zapman449 
Finally fixed #545

**Comment by kotakanbe** [comment]
Please re-open if you are still in trouble.
